src/Template/User/settings.ctp

```diff
@@ -172,21 +172,21 @@ $this->set('title_for_layout', $this->Pages->formatTitle(__('Settings')));
             </md-list-item>
 
             <md-list-item>
-                <?php $hideRandomSentence = $userSettings->settings['hide_random_sentence']; ?>
+                <?php $showRandomSentence = $userSettings->settings['hide_random_sentence']; ?>
                 <md-checkbox
-                    ng-false-value="0"
-                    ng-true-value="1"
-                    ng-model="hideRandomSentence"
-                    ng-init="hideRandomSentence = <?= $hideRandomSentence ?>"
+                    ng-false-value="1"
+                    ng-true-value="0"
+                    ng-model="showRandomSentence"
+                    ng-init="showRandomSentence = <?= $showRandomSentence == false ? 0 : $showRandomSentence ?>"
```

Comment:
Casting the value into an integer is more concise and secure: `(int)$showRandomSentence`.